test/sjcl-ecdsa-recoverablepublickey-test.js

```diff
@@ -43,7 +43,7 @@ describe('ECDSA signing with recoverable public key', function(){
 
         var pub_val_point = secret_key._curve.G.mult(secret_key._exponent);
         var public_key = new sjcl.ecc.ecdsa.publicKey(curve, pub_val_point);
-        var hash = hash_function(message);
+        var hash = sjcl.bitArray.clamp(hash_function(message), secret_key._curveBitLength);
```

Comment:
Yeah, I created a ticket for extracting it a few weeks ago: https://ripplelabs.atlassian.net/browse/RLJS-358